scripts/start-full-kessel.sh

```
@@ -30,7 +40,26 @@ else
 
+# Fetch RBAC role definitions from stage configmap (replaces baked-in definitions
+# that include poisoned approval_* roles not in the SpiceDB schema)
+RBAC_DEFS_DIR="${COMPOSE_DIR}/configs/rbac-role-definitions"
+mkdir -p "${RBAC_DEFS_DIR}"
+if [ -n "${RBAC_CONFIG_FILE}" ]; then
+  echo "Using local RBAC config: ${RBAC_CONFIG_FILE}"
+  RBAC_CONFIG_SRC="${RBAC_CONFIG_FILE}"
+else
+  RBAC_CONFIG_URL="${RBAC_CONFIG_URL:-https://raw.githubusercontent.com/RedHatInsights/rbac-config/refs/heads/master/_private/configmaps/stage/rbac-config.yml}"
+  RBAC_CONFIG_SRC=$(mktemp)
+  trap "rm -f ${RBAC_CONFIG_SRC}" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)